test/e2e/specs/media/settings__media.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * @group jetpack-remote-site
+ * @group jetpack-wpcom-integration
+ */
+
+import {
+	DataHelper,
+	TestAccount,
+	SidebarComponent,
+	WritingSettingsPage,
+	envVariables,
+	getTestAccountByFeature,
+	envToFeatureKey,
+	WpAdminMediaSettingsPage,
+} from '@automattic/calypso-e2e';
+import { Page, Browser } from 'playwright';
+
+declare const browser: Browser;
+
+// We care about these settings in all Jetpack sites, but they are in a different place
+// depending on whether the site is a WPCOM site or a remote site.
+// So, we have to do some condition case logic here.
+describe( DataHelper.createSuiteTitle( 'Jetpack Settings: Media' ), function () {
+	let page: Page;
+
+	const accountName = getTestAccountByFeature( envToFeatureKey( envVariables ) );
+
+	beforeAll( async function () {
+		page = await browser.newPage();
+
+		const testAccount = new TestAccount( accountName );
+		await testAccount.authenticate( page );
+	} );
+
+	if ( envVariables.JETPACK_TARGET === 'remote-site' ) {
```

Comment:
**comment:non-blocking** - I am not the biggest fan of conditional logic at the block/step level, but it is also true that there is no cleaner way of doing this.

The most commonly cited way of achieving this is to use the `skipIf` implementation that I added some time ago. That in itself feels a bit like a kludge.